okhttp/src/main/java/okhttp3/CertificatePinner.kt

```diff
@@ -165,11 +166,11 @@ class CertificatePinner internal constructor(
 
       for (pin in pins) {
         when (pin.hashAlgorithm) {
-          "sha256/" -> {
+          "sha256" -> {
```

Comment:
I think the body of this for loop to be a method, `Pin.matchesCertificate(certificates: List<Certificate>)`.